pom.xml

```diff
@@ -74,10 +74,9 @@
         <powerauth-crypto.version>1.5.0-SNAPSHOT</powerauth-crypto.version>
 
         <!-- Library Versions -->
-        <guava.version>31.1-jre</guava.version>
         <pushy.version>0.15.2</pushy.version>
         <springdoc-openapi.version>1.6.14</springdoc-openapi.version>
-        <google-api-client.version>1.35.1</google-api-client.version>
+        <google-api-client.version>1.35.2</google-api-client.version>
```

Comment:
The latest patch for 1.x.x, let's postpone the upgrade to 2.2.0 for the next release.